docs/guide/security-authorization.md

```diff
@@ -487,6 +491,37 @@ public function behaviors()
 If all the CRUD operations are managed together then it's a good idea to use a single permission, like `managePost`, and
 check it in [[yii\web\Controller::beforeAction()]].
 
+You can also use rules from the [[yii\filters\AccessControl|AccessControl]] filter. For that you specify the
+[[yii\filters\AccessRule::roleParams|roleParams]] that you need to pass to the [[yii\filters\AccessRule|AccessRule]]:
+
+
+```php
+use yii\filters\AccessControl;
+
+class PostsController extends Controller
+{
+    public function behaviors()
+    {
+        return [
+            'access' => [
+                'class' => AccessControl::className(),
+                'rules' => [
+                    [
+                        'actions' => ['update'],
+                        'roles' => ['updatePost'],
+                        'roleParams' => function() {
```

Comment:
It would be nice to have an example without closure too, to make it clear it is possible.